shell/ci/release/docker.sh

```diff
@@ -171,6 +165,26 @@ build_and_push_image() {
         "$buildContext"
     )
   fi
+
+  # Tag the images for devenv
+  # For primary container, the image should be built with gcr.io/outreach-docker/$APPNAME:latest
+  # For dependency container, the image should be built with gcr.io/outreach-docker/$APPNAME/$image:local 
+  if [[ -n $DEVENV_DOCKER_BUILD ]]; then
+    info "Tagging the images for devenv"
+    if [[ "$image" != "$APPNAME" ]]; then
+      devenv_image="gcr.io/outreach-docker/$APPNAME/$image"
+      (
+        set -x
+        docker tag "$image":latest "$devenv_image":local
+      )
+    else
+      devenv_image="gcr.io/outreach-docker/$image"
```

Comment:
`$remote_image_name` should suffice instead of hardcoding here, right?

Reply:
ahh, right. Good reminding.